portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,11 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+async function fetchHelloWorld() {
+	document.getElementById("inner-content").style.display = "block";
+	const response = await fetch('/data');
+	const hello = await response.text();
+	document.getElementById("my-info").innerText = hello;
```

Comment:
nit - per style guide, use [single quotes](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes) instead of double quotes when quoting strings.

Reply:
thanks! will do!